### ISSUE TITLE
Bug - Blockquote - missing top padding

### ISSUE THREAD

**Issue by SophieAmnesty** [amnestywebsite/humanity-theme#294]
**Describe the bug**  
The top of the blockquote block is touching the paragraph above. 

**To Reproduce**  
1. create new post 
2. add paragraph 
3. add blockquote underneath
4. view on front end - see no space between

**Expected behaviour**  
There should be spacing in between the paragraph and the blockquote.

**Actual behaviour**  
The two blocks are too close.

**Link to a page where the bug is reproduced in isolation (optional)**  
https://www.amnesty.org/en/latest/news/2024/07/europe-sweeping-pattern-of-systematic-attacks-and-restrictions-undermine-peaceful-protest/ 

**Screenshots**  
![image](https://github.com/amnestywebsite/humanity-theme/assets/99269799/ff91312a-fadc-4ee4-a96b-f1925a93a39e)

**Platform:**  
https://www.whatsmybrowser.org/b/X0NLV

**Comment by AmnestyAM** [comment]
FYI I went rogue and did this on .org will remove when we do a release 

![image](https://github.com/user-attachments/assets/7042a63d-6480-4ac0-bade-345e65b59d3e)

**Comment by madhusudhand** [comment]
The PR #298 is merged and this could be closed as resolved.

**Comment by jaymcp** [comment]
Thanks @madhusudhand
Although the PR has been merged to `develop`, there are still a few steps that need to take place before we can consider the issue complete. Here's a quick rundown of our workflow:

1. a new branch is created from `main` 
2. work is completed on the branch
3. the branch is PR'd
4. a reviewer tests and code reviews
5. once approved, the PR is merged into `develop`
6. `develop` deploys to our testing environment
7. testing is performed against the PR on the testing environment
8. once testing has passed, the PR branch is then merged into `staging`
9. `staging` deploys to our User Acceptance Testing (UAT) environment
10. UAT is performed against the PR on the UAT environment
11. once UAT is passed, the ticket(s) associated with the PR is added to a Release Candidate ticket ([example](https://github.com/amnestywebsite/humanity-theme/issues/282)) and is closed as complete
12. a release branch is created from `main`, into which all approved PRs are merged
13. once the Release Candidate has been signed off for release, the release branch is PR'd into `main` for final checks prior to release
14. the release branch is merged into `main`
15. the merge to `main` triggers a draft release build in GitHub
16. once the draft build has passed, we can then publish the GitHub release

At each stage of the process, we update the status field of the ticket in the GitHub project, so that we know what the status of each ticket is:

![](http://bigbite.im/i/wdRAy5+)

You can view the status board on the GitHub project here: https://github.com/orgs/amnestywebsite/projects/3/views/2

**Comment by AmnestyAM** [comment]
LGTM
